packages/components/src/components/SquiggleErrorAlert.tsx

```diff
@@ -35,24 +38,28 @@ const WithHeader: FC<PropsWithChildren<{ header: string }>> = ({
   children,
 }) => (
   <div>
-    <div className="font-medium">{header}</div>
-    <div className="ml-4">{children}</div>
+    <div className="text-sm font-medium mb-1 text-gray-700">{header}</div>
+    <div className="ml-1">{children}</div>
   </div>
 );
 
 const StackTraceFrame: FC<{ frame: SqFrame }> = ({ frame }) => {
   const location = frame.location();
+  const name = frame.name();
   return (
-    <div>
-      {frame.name()} {location ? <LocationLine location={location} /> : ""}
+    <div className="grid grid-cols-[1fr_3fr] gap-x-2">
+      <div className="text-gray-500 font-mono font-medium truncate">
+        {name !== "" && name}
```

Comment:
Why not just `{name}`? I can't think of a situation where it can be empty, and the result is identical either way.

Reply:
Fair, this is an artifact of some previous stuff, can fix.